cpp/src/gandiva/precompiled/extended_math_ops_test.cc

```diff
@@ -87,6 +87,22 @@ TEST(TestExtendedMathOps, TestLogWithBase) {
   EXPECT_EQ(context1.has_error(), false);
 }
 
+TEST(TestExtendedMathOps, TestRound) {
+  EXPECT_EQ(round_int32_int32(85, -1), 90);
```

Comment:
@sagnikc-dremio this was what I referring to on my previous testing comments, do these seem reasonable?

Reply:
Yes, the result should be -90. Thanks for pointing it out. Looking into the test failures.